multi-cluster-spring-boot/.applier/inventory-pre-dev/group_vars/seed-hosts.yml

```diff
@@ -0,0 +1,6 @@
+openshift_cluster_content:
```

Comment:
I don't know that this is necessary. We already have a method to create a promoter SA in prod, and a token secret for that SA in dev.